flink-metrics/flink-metrics-influxdb/src/test/java/org/apache/flink/metrics/influxdb/InfluxdbReporterTest.java

```diff
@@ -96,33 +96,36 @@ public void testMetricRegistration() throws Exception {
 
 	@Test
 	public void testMetricReporting() throws Exception {
-		MetricRegistryImpl metricRegistry = createMetricRegistry();
+		String retentionPolicy = "one_hour";
+		MetricRegistryImpl metricRegistry = createMetricRegistry(retentionPolicy);
 		try {
 			String metricName = "TestCounter";
 			Counter counter = registerTestMetric(metricName, metricRegistry);
 			counter.inc(42);
 
 			stubFor(post(urlPathEqualTo("/write"))
-				.willReturn(aResponse()
-					.withStatus(200)));
+					.willReturn(aResponse()
+							.withStatus(200)));
 
 			InfluxdbReporter reporter = (InfluxdbReporter) metricRegistry.getReporters().get(0);
 			reporter.report();
 
 			verify(postRequestedFor(urlPathEqualTo("/write"))
-				.withQueryParam("db", equalTo(TEST_INFLUXDB_DB))
-				.withHeader("Content-Type", containing("text/plain"))
-				.withRequestBody(containing("taskmanager_" + metricName + ",host=" + METRIC_HOSTNAME + ",tm_id=" + METRIC_TM_ID + " count=42i")));
+					.withQueryParam("db", equalTo(TEST_INFLUXDB_DB))
+					.withQueryParam("rp", equalTo(retentionPolicy))
+					.withHeader("Content-Type", containing("text/plain"))
+					.withRequestBody(containing("taskmanager_" + metricName + ",host=" + METRIC_HOSTNAME + ",tm_id=" + METRIC_TM_ID + " count=42i")));
```

Comment:
The code formatting has changed here and 10 lines above.

Otherwise, I don't have anything to add any more.
@mans2singh, you can resolve my other comments if you want.